packages/node_modules/@ciscospark/redux-module-media/src/actions.js

```diff
@@ -239,19 +260,25 @@ function constructCallState(call) {
 }
```

Comment:
Should we remove these remote media objects? I think at least some of these don’t do anything anymore

Reply:
I don't see anything that isn't used.